tests/PregTest.php

```diff
@@ -51,9 +51,9 @@ public function providePatternValidationCases()
     {
         return [
             'invalid_blank' => ['', null, PregException::class],
-            'invalid_open' => ["\1", null, PregException::class, "'\1' found"],
-            'valid_control_character_delimiter' => ["\1\1", 1],
-            'invalid_control_character_modifier' => ["\1\1\1", null, PregException::class, ' Unknown modifier '],
+            'invalid_open' => ['#', null, PregException::class, "'#' found"],
+            'valid_control_character_delimiter' => ['##', 1],
+            'invalid_control_character_modifier' => ['###', null, PregException::class, ' Unknown modifier '],
```

Comment:
Due to this: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5314/checks?check_run_id=1491295757#step:9:1012

Reply:
Never mind, it just works now!